src/lib/search/searchHelpers.ts

```diff
@@ -137,3 +139,19 @@ export function mapIndexToMessage(index: SearchableIndex) {
       return "";
   }
 }
+
+/**
+ * WARNING: This is non-reversible.
```

Comment:
> `WARNING: This is non-reversible.`

Does this matter? In any case, it's reversible now.

Reply:
No it doesn't really matter, but just a heads up that we cannot convert back to Prisma ID given Meili ID. But doesn't matter now